app/scripts/controllers/preferences.js

```diff
@@ -671,8 +672,8 @@ export default class PreferencesController {
    *
    *
    */
-  _subscribeProviderType() {
-    this.network.providerStore.subscribe(() => {
+  _subscribeToNetworkDidChange() {
```

Comment:
As far as I can tell, this change is functionally equivalent, as the NETWORK_DID_CHANGE event is only emitted at the time that the providerStore is updated.